grails-app/views/admin/adminReports.gsp

```diff
@@ -151,6 +166,36 @@
 
     });
 
+    // Activates knockout.js
```

Comment:
Changes to the view model:
1) Move it into a javascript file.
2) Name it something more meaningful (e.g. ManagementUnitReportSelectorViewModel)
3) Delete the "this.yourName"?
4) The view model should include the generateMuReport function
5) The buttons that trigger the user report should be using knocking bindings to the view model rather than jquery
6) Write a javascript unit test for your new view model.
7) Write a simple functional test to navigate to the page, generate a MU report and ensure an email is sent.